README.md

```diff
@@ -32,3 +32,12 @@ print(parsed_md)
 - pages_per_split (int, optional): Number of pages per split for chunking. Defaults to 4.
 - max_threads (int, optional): Maximum number of threads for parallel processing. Defaults to 4.
 - **kwargs: Additional arguments for the parser.
+
+## Benchmark
+
+| Rank | Model | Similarity | Time (s) |
```

Comment:
On attempting to re-run the benchmark,
This is what i got, a bit of a variance.
In future evals, maybe we also aggregate across a few runs (aggregate over 3 runs)?

| Rank | Model          | Similarity | Time (s) | 
|------|-------------|------------|----------|
| 1 | gpt-4o | 0.814 | 19.28 |
| 2 | gemini-1.5-pro | 0.789 | 15.28 |
| 3 | gemini-1.5-flash | 0.731 | 4.26 |
| 4 |  gpt-4o-mini | 0.699 | 16.54 |

Reply:
Added